scripts/update-index.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# This script update 'docs/index.md' with 'README.md' 
+#  changing github/learn-ocaml/docs URLs to relative path.
```

Comment:
English nitpick + add a quick motivation:

```suggestion
# This script updates 'docs/index.md' from 'README.md',
# and changes github.com/…/learn-ocaml/…/docs URLs to relative paths,
# so that github.io pages work independently of github.com src files.
```